Sources/TuistServer/Client/ServerClientAuthenticationMiddleware.swift

```diff
@@ -67,7 +59,7 @@ struct ServerClientAuthenticationMiddleware: ClientMiddleware {
         var request = request
 
         /// Cirrus environments don't require authentication so we skip in these cases
-        if envVariables[Constants.EnvironmentVariables.cirrusTuistCacheURL] != nil {
+        if envVariables["CIRRUS_TUIST_CACHE_URL"] != nil {
```

Comment:
We should move `TuistSupport.Constants` enum to its own module, `Constants`. I didn't want to do too many changes here, so I used a raw string here since we don't really use this much right now anyway.